asf_public_discourse_home_decarbonisation/analysis/mse/initial_text_analysis_category_data.py

```diff
@@ -0,0 +1,442 @@
+"""
+This script performs an initial text analysis on Money Saving Expert data, by:
+- Computing the frenquency os specific words and n-grams of interest;
+- Identiyfying top words and n-grams;
+
+Plots are saved in the folder `asf_public_discourse_home_decarbonisation/outputs/figures/mse/`.
+
+To run this script:
+`python asf_public_discourse_home_decarbonisation/analysis/mse/initial_text_analysis_category_data.py`
+
+To change category or date/time range, use the following arguments:
+`python asf_public_discourse_home_decarbonisation/analysis/mse/initial_text_analysis_category_data.py --category <category> --collection_date_time <collection_date_time>`
+where
+<category>: category or sub-forum to be analysed, e.g. "energy", "lpg-heating-oil-solid-other-fuels", etc.
```

Comment:
as you mention, it does not work for the energy category, at least for myself when I try to run for energy.